build.gradle

```diff
@@ -51,6 +52,28 @@ jar.dependsOn generatePom
 
 /*
 =========================================================
+   Generate Java Api
+=========================================================
+ */
+task generateJava(type:Exec) {
+    def srcDir = "src/main/thrift"
+    def targetDir = new File("src/main/java")
+    targetDir.mkdirs()
+
+    project.file(srcDir).listFiles(new FilenameFilter() {
+        @Override
+        public boolean accept(File dir, String name) {
+            return name.matches(".*\\.(thrift)");
```

Comment:
Not sure why you have parens, it seems like it would theoretically add a small amount of grouping logic that you don't need, but it works and I don't really care.